packages/ui-components/src/components/Table/ExpandableTable.js

```diff
@@ -4,13 +4,24 @@
  */
 
 import React from 'react';
+import styled from 'styled-components';
 import PropTypes from 'prop-types';
 import { Table } from './Table';
 import { ExpandableTableBody } from './ExpandableTableBody';
 
-export const ExpandableTable = React.memo(({ Body, SubComponent, ...tableProps }) => {
-  const TableBody = tableBodyProps => <Body {...tableBodyProps} SubComponent={SubComponent} />;
-  return <Table Body={TableBody} {...tableProps} />;
+const StyledTable = styled(Table)`
+  th:last-child,
+  td:last-child {
+    padding-right: 65px;
+  }
```

Comment:
Those styles are repeated in `ExpandableTableBody` and `ExpandableTableHeader`. I do realise that sometimes it's better to repeat a styling value rather than over-abstract its definition. Thus, I have not suggestion to make regarding this, just flagging it